supertokens_python/recipe/session/session_class.py

```diff
@@ -55,7 +62,7 @@ async def update_session_data(
 
     async def update_access_token_payload(
         self,
-        new_access_token_payload: Dict[str, Any],
+        new_access_token_payload: Union[Dict[str, Any], None],
```

Comment:
I don't think this should be `Union[Dict[str, Any], None],`. Just `Dict[str, Any],` (I know in node SDK, it was changed to any | undefined, but that might be a mistake as well)

Reply:
Okay. done.